src/main.py

```diff
@@ -0,0 +1,36 @@
+from collection import HTTPPacketCollector
+from statistics_management import StatisticsManager
+from alerting import AlertManager
+from argparse import ArgumentParser
+from time import sleep
+
+
+def main(args):
+    alert_timer = 0  # every 12 10 second waits we'll also check for the 2m mark
+    packet_collector = HTTPPacketCollector()
+    packet_collector.start_packet_collection(args.interface if args.interface else None)
+    alerts = AlertManager(int(args.alert_trigger_threshold))
+    statistics = StatisticsManager()
+    statistics.show_all_statistics()
+    while True:
+        sleep(10)
+        alert_timer += 1
+        statistics.calculate_statistics(packet_collector.collection)
+        statistics.show_all_statistics()
+        statistics.clear_recent_statistics()
```

Comment:
Can this be moved into a single method in StatisticsManager since they're all related?

If these steps change, then the change is localized.